manager/src/main/java/com/redhat/service/bridge/manager/api/user/BridgesAPI.java

```diff
@@ -52,26 +61,57 @@
     @Inject
     JsonWebToken jwt;
 
+    @APIResponses(value = {
+            @APIResponse(description = "Success.", responseCode = "200",
+                    content = @Content(mediaType = MediaType.APPLICATION_JSON, schema = @Schema(type = SchemaType.OBJECT, implementation = BridgeListResponse.class))),
+            @APIResponse(description = "Bad request.", responseCode = "400", content = @Content(mediaType = MediaType.APPLICATION_JSON)),
+            @APIResponse(description = "Unauthorized.", responseCode = "401"),
+            @APIResponse(description = "Forbidden.", responseCode = "403")
+    })
+    @Operation(summary = "Get the list of ingresses", description = "Get the list of ingresses for the authenticated user.")
     @GET
     public Response getBridges(@Valid @BeanParam QueryInfo queryInfo) {
         System.out.println(jwt.getSubject());
         return Response.ok(ListResponse.fill(bridgesService
                 .getBridges(identityResolver.resolve(jwt), queryInfo), new BridgeListResponse(), bridgesService::toResponse)).build();
     }
 
+    @APIResponses(value = {
+            @APIResponse(description = "Accepted.", responseCode = "202",
+                    content = @Content(mediaType = MediaType.APPLICATION_JSON, schema = @Schema(type = SchemaType.OBJECT, implementation = BridgeResponse.class))),
+            @APIResponse(description = "Bad request.", responseCode = "400", content = @Content(mediaType = MediaType.APPLICATION_JSON)),
+            @APIResponse(description = "Unauthorized.", responseCode = "401"),
+            @APIResponse(description = "Forbidden.", responseCode = "403")
```

Comment:
Does it have sense to mention also HTTP 500?
`Internal Server Error`
```
{
  "kind": "Error",
  "id": "9",
  "href": "/api/v1/errors/9",
  "code": "OPENBRIDGE-9",
  "reason": "There was an internal exception that is not fixable from the user. Please open a bug with all the information you have, including the name of the exception com.redhat.service.bridge.infra.exceptions.definitions.platform.InternalPlatformException and the message Timeout reached while creating topic and granting access to topic 'ob-42d79b80-6faa-45ed-a308-769daa91ca60'"
}
```

Reply:
done